src/bot.py

```diff
@@ -35,10 +35,15 @@
     ]
 )
 
-hour_in_minutes = 3600
-cooldown = config['hours_to_wait'] * hour_in_minutes
+mininterval = config['mininterval']
+maxinterval = config['maxinterval']
+
+cooldown = randint(mininterval, maxinterval) * 24 * 60 *60
+#converting the input of minimum and maximum interval from days, to seconds

```

Comment:
`# converts the random value from days to seconds`

That's a bit cleaner. Make sure to keep the space for legibility.

Reply:
my bad! I'll be updating that shortly